website/docs/reference/widgets/chart.mdx

```diff
@@ -0,0 +1,412 @@
+
+```js
+[
+  {
+    "x": "Product1",
+    "y": 20000
+  },..
+]
+```
+
+In this format, **`x`** denotes the **label** and **`y`** denotes the **value**. It's important to provide a value when displaying data to ensure that the chart can render correctly. Please note that the JSON format provided is only suitable for basic charts
+
+Appsmith offers several quick and easy ways to represent your data. With the Chart Type property, you can select one of the following types to visualize your data:
+* Line Charts, 
+* Bar Charts, 
+* Pie Charts, 
+* Column Charts, 
+* Area Charts, and  
+* [Custom Charts](#custom-charts)
+
+
+### Display static data
+
+To display static data, you can enter the desired data directly into the `Series Data property`, and it remains the same until manually updated or edited. This method is useful when you need to display fixed information, such as trends or comparisons, that don't change based on user interactions. 
+
+```js
+[
+  {
+    "x": "2021",
+    "y": 200000
+  },
+  {
+    "x": "2022",
+    "y": 320000
+  }
+]
+```
+
+### Display data dynamically
+
+You can display dynamic data by fetching it from APIs or databases. This can be done by using binding values to **Series Data property**. If you need to transform the incoming data to match this format, you can use **JSObject** to perform the transformation before binding it to the chart widget.
+
+For example, lets say you have a PostgreSQL database containing information about assets in your organization, such as their types and the date they were purchased. You can create a query that groups the assets by type and counts how many there are of each type, like this:
+
+```sql
+select type as x, count (*) as y from asset group by type
+```
+
+This query returns data in the required JSON format, where `x` represents the asset type and `y` represents the count of assets of that type. Next, in the chart widget's **Series** property, add `{{assetquery.data}}` to bind the data returned by the query to the chart. This displays a dynamic chart that updates as the data in the database changes.
+
+### Display multiple data columns
+
+If you want to display multiple data columns on a chart, you can do so by using the **Add Series** property. This feature allows you to populate the chart with different sets of data and customize various aspects of its appearance, such as colors and labels, to create a visually appealing and informative chart.
+
+For example, if you have data on the number of males and females who voted in each year of a political election, you can use the Chart Series property to display both sets of data on the same chart. This would allow you to compare the number of male and female voters over time.
+
+Check this [sample app](https://app.appsmith.com/applications/620b668f6b4b1e154a3c2418/pages/620b8bc56b4b1e154a3c2688) to learn how to produce a dual-axis chart combining columns, lines, and areas.
```

Comment:
this sample is for custom charts. Let's give an example. even a screenshot showing this example will do